data/json/mapgen/ranch_camp.json

```diff
@@ -2478,475 +680,180 @@
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%...................................................................%.",
+        "..%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%.",
+        "........................................................................",
+        "........................................................................"
+      ],
+      "palettes": [ "ranch_camp" ]
+    },
+    "om_terrain": [ [ "ranch_camp_73", "ranch_camp_74", "ranch_camp_75" ] ],
+    "type": "mapgen",
+    "weight": 250
+  },
+  {
+    "method": "json",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        ".....______________.......%.....................",
+        "..._________________......%.....................",
+        "...__________________/....%.....................",
+        "//.__________________/./.6y.....................",
+        "./.__________________./../G.....................",
+        "///__________________../..G.....................",
+        "///__________________././.G.....................",
+        "./.__________________..//.G.....................",
+        "//.________,,________.//./G.....................",
+        ".//________,,________.....G.....................",
+        ".//________,,________./...y.....................",
+        "/./__________________.....%.....................",
+        "//.________,,________.....%.....................",
+        "../________,,________.....%.....................",
+        "...________,,________.....%.....................",
+        "...__________________.....%.....................",
+        "...________,,________.....%.....................",
+        "...________,,________.....%.....................",
+        "...________,,________.....%.....................",
+        "...__________________.....%.....................",
+        "...________,,________.....%.....................",
+        "...________,,________.....%%%%%%%%%%%%%%%%%%%%%%",
+        "...________,,________...........................",
+        "...__________________..........................."
+      ],
+      "palettes": [ "ranch_camp" ]
+    },
+    "om_terrain": [ [ "ranch_camp_77", "ranch_camp_78" ] ],
```

Comment:
`ranch_camp_76` is missing

![image](https://user-images.githubusercontent.com/16213433/57771985-fb467880-771c-11e9-86c3-d6ac7d73b8a1.png)